vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -61,7 +61,7 @@ Use `ent help` to review the list of available commands
 ~~~~~~~~~~~~~~~~~~~
 
 > Essentials:
-  - Activate using:  ~/.entando/ent/v6.3.2/cli/v6.3.2/activate
+  - Activate using:  ~/.entando/ent/v7.0.0/cli/v7.0.0/activate
```

Comment:
@nshaw unsure if this was correct for the cli

Reply:
Looks good